File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.java
class Solution {
    public int maxCoins(int[] piles) {
        Arrays.sort(piles);
        int res = 0;

        for (int i = piles.length / 3; i < piles.length; i += 2) {
            res += piles[i];
        }

        return res;        
    }
}

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.js
/**
 * @param {number[]} piles
 * @return {number}
 */
var maxCoins = function(piles) {
    piles.sort((a, b) => b - a)
    let total = 0, count = piles.length / 3, index = 1

    while (count-- > 0) {
        total += piles[index]
        index += 2
    }
    
    return total
};